code/modules/tgui_input/alert.dm

```diff
@@ -22,7 +22,7 @@
 	// A gentle nudge - you should not be using TGUI alert for anything other than a simple message.
 	if(length(buttons) > 3)
 		log_tgui(user, "Error: TGUI Alert initiated with too many buttons. Use a list.", "TguiAlert")
-		return tgui_input_list(user, message, title, buttons, timeout, autofocus)
+		return tgui_input_list(user, message, title, buttons)
```

Comment:
this (mob/user, message, title = "Select", list/items, default, timeout = 0) so you can still pass timeout

Reply:
> this (mob/user, message, title, list/items, default, timeout) so you can still pass timeout

What should i pass to default argument?

Reply:
> this (mob/user, message, title, list/items, default, timeout) so you can still pass timeout

What should i pass to default argument? Because no one used that. And It's bad idea because many have different 'no' choices like 'Cancel', 'None', etc.

Reply:
You can ignore it it you want and just do 
return tgui_input_list(user, message, title, buttons, timeout=timeout)